docs/api/createStore.md

```diff
@@ -50,7 +50,7 @@ console.log(store.getState())
 
 - Don't create more than one store in an application! Instead, use [`combineReducers`](combineReducers.md) to create a single root reducer out of many.
 
-- It is up to you to choose the state format. You can use plain objects or something like [Immutable](https://facebook.github.io/immutable-js/). If you're not sure, start with plain objects.
+- It is up to you to choose the state format. You can use plain objects or something like [Immer](https://github.com/immerjs/immer). If you're not sure, start with plain objects.
```

Comment:
:hand: This paragraph should be rewritten, along the lines of "Redux state is normally plain JS objects and arrays", and maybe just drop the "other formats" aspect.